src/Administration/Resources/app/administration/src/module/sw-settings/page/sw-settings-index/index.js

```diff
@@ -59,6 +76,27 @@ export default {
     },
 
     methods: {
+        /**
+         * @deprecated tag:v6.8.0 - Will be removed without replacement
+         */
+        async getUserConfig() {
+            const response = await this.userConfigService.search(['settings.hideMigratingScalesUnitBanner']);
+            this.hideSettingRenameBanner = !!response.data['settings.hideMigratingScalesUnitBanner']?.data;
+        },
+
+        /**
+         * @deprecated tag:v6.8.0 - Will be removed without replacement
+         */
+        async onCloseUnitBanner() {
+            this.hideSettingRenameBanner = !this.hideSettingRenameBanner;
+
+            await this.userConfigService.upsert({
```

Comment:
await this.userConfigService.upsert({
                'settings.hideMigratingScalesUnitBanner': true,
            });

Reply:
I mean, it's always true here